apps/web/src/pages/EmployeeProfile/components/CareerObjective/CareerObjectiveSection.tsx

```diff
@@ -554,6 +602,35 @@ const CareerObjectiveSection = ({
                     disabled={fetching}
                   />
                 ) : null}
+                <Checkbox
+                  boundingBox
+                  boundingBoxLabel={intl.formatMessage(
+                    employeeProfileMessages.seniorManagementStatus,
+                  )}
+                  id="isCSuiteRole"
+                  label={intl.formatMessage(
+                    employeeProfileMessages.isChiefDeputyCSuiteRole,
+                  )}
+                  name="isCSuiteRole"
+                />
+                {watchIsCSuiteRole === true ? (
```

Comment:
This could just be the following I think

```tsx
{watchIsCSuiteRole && (
  <Select  />
)}
```

Reply:
Yeah, think so. Simplifying it https://github.com/GCTC-NTGC/gc-digital-talent/pull/12975/commits/e94ad56dbe1cf5ce8cfdd54c18942764c30482ba